### ISSUE TITLE
Keybind cast on frame

### ISSUE THREAD

**Issue by Dogyy666** [Snakybo/Clicked#222]
## Is your feature request related to a problem?
Im trying to make macro to bind my spell into VohDo panel. 

## Describe the solution you'd like
Lets say we have raid. In VuhDo or any other addon like this i can create grp of players in that that i want to focus on. lets say 4 players that i monitor all the time on separate frame group and i want to make macro to cast spell on them without mouseovering. Lets say something like /cast [@VuhDo-1] spellname. but its impossible to make macro without knowing this frame name ye?. something like u have default in blizzard party /cast [@party1] spellname. 
Maybe its possible to just click on frame to make macro for this specific frame?

## Describe alternatives you've considered
Maybe if we can get somehow framename we can add it to macro manualy
## Additional context
On left side is just regular default blizzard raidframes on right side its my created group of player that i want to track in raid and just instead of using mouseover i want to just make 5 keysbinds every assigned to one player on this smart frame i created.
https://cdn.discordapp.com/attachments/1196860740875866172/1281267122550341642/8E168748-0220-402E-9734-2AADE7739F68.png?ex=66db187e&is=66d9c6fe&hm=6340cb5e1a1f6661dea5feb00f53a07b8a1a5106a3e78d15ac1126bb251a9653&

**Comment by Snakybo** [comment]
Hm, not sure I fully understand — this should work as-is if you create a spell binding and enable "Unit frame" on the targets page. Alternatively, if you have to use a custom macro for whatever reason, use `[@mouseover]` and set the activation mode to "Unit frame".

**Comment by Dogyy666** [comment]
as u can see on right side of the screen i have 5 players that i want to focus on from raid. how i can make macro for them if i dont know unit frame name? I want to be able to cast spells on them without using mouseover. As u can see on screenshot player named "Largakh" is 1st on those frames i created in VuhDo. So i need to make macro to cast spell on him without using mouse over just i press lets say "prescience" and im casting Prescience on him. then i need secound macro to cast on player "Korgath". But i dont wanna hardmacro those names likes /cast [@Korgath] Prescience i want something that allows me to cast spell on player thats on this frame whatever his name is. but i cant see whats this frame name. Is it possbile to get name of those frames to macro it? havnt seen those options anywhere i can see only ( https://cdn.discordapp.com/attachments/1196860740875866172/1281502614168535082/32652694-B5BB-4FA2-80EC-4B182588AEBC.png?ex=66dbf3d0&is=66daa250&hm=0b15098e612b159f50dde600791370b4d9ed675896157f3dfbd4c51c71cdf904& ) 
so maybe theres option to just click on frame to make macro for it or addon to detect those frames that are not default or any other option to macro it.

**Comment by Snakybo** [comment]
Right, sorry it took a while 😅 unfortunately I don't think that's possible in the game, as far as I know inside of VuhDo those are just aliases for `@party1`, `@raid6`, etc. so the only way to achieve this is to create bindings for those specific units, with the downside of having to make sure your group is in the same order every time. Other than that I don't think this is possible.
